src/model_repository_manager.h

```diff
@@ -81,7 +81,6 @@ class ModelRepositoryManager {
     std::string model_name_;
     Status status_;
     bool checked_;
-    // FIXME
```

Comment:
was this not needed? or is there some corresponding ticket?

Reply:
Looks like this was added by @GuanLuo a while back here: https://github.com/triton-inference-server/server/commit/09271f9c4d4d935bd9667dd2be2208d2dd5effd0

@GuanLuo any context?